code/modules/mob/living/living.dm

```diff
@@ -1274,7 +1274,7 @@ Thanks.
 	return
 
 //same as above
-/mob/living/pointed(atom/A as mob|obj|turf in view(get_turf(src)))
+/mob/living/point(atom/A as mob|obj|turf in view(get_turf(src)))
```

Comment:
The Ref might be lying again but if it's not, just change `get_turf(src)` to `src` instead
That *should* make it so that you can't use it on things you can't see

Reply:
Alternative solution is to just eat the marginal performance loss and make it an atom verb instead

Reply:
>The Ref might be lying again but if it's not, just change get_turf(src) to src instead

I think I changed that a few months back so that pAIs could point at things.

Reply:
What the Ref says you should do there is `client` but I'm pretty sure we already found out that doesn't work